tests/test_environment_indexentries.py

```diff
@@ -69,8 +70,9 @@ def test_create_pair_index(app):
                                       ('ёлка', [('', '#index-5')]),
                                       ('Ель', [('', '#index-4')])],
                                      None])])
-    assert index[6] == ('Е', [('ёлка', [[], [('Sphinx', [('', '#index-5')])], None]),
-                               ('Ель', [[], [('Sphinx', [('', '#index-4')])], None])])
+    assert index[6] == ('Е',
+                        [('ёлка', [[], [('Sphinx', [('', '#index-5')])], None]),
+                         ('Ель', [[], [('Sphinx', [('', '#index-4')])], None])])
```

Comment:
I posted https://github.com/sphinx-doc/sphinx/pull/8430 as a fix of these indentation warning.